Formula/stacc.rb

```diff
@@ -0,0 +1,32 @@
+# typed: false
+# frozen_string_literal: true
+
+# This file was generated by GoReleaser. DO NOT EDIT.
+class Stacc < Formula
+  desc "Software used to communicate with Stacc cloud environments"
+  homepage "https://github.com/stacc/stacc-CLI"
+  version "0.1.0"
+  license "Apache-2.0"
+  bottle :unneeded
+
+  if OS.mac?
+    url "https://github.com/stacc/stacc-CLI/releases/download/v0.1.0/stacc_0.1.0_Darwin_x86_64.tar.gz"
+    sha256 "97be81a0c28dc49a9b9301938443561f5dca9c3f79369c9c422132995b97f7de"
+  end
+  if OS.linux? && Hardware::CPU.intel?
+    url "https://github.com/stacc/stacc-CLI/releases/download/v0.1.0/stacc_0.1.0_Linux_x86_64.tar.gz"
+    sha256 "bb005646963a54e231e0f4ebba1b630353ea546771b3e33d3e0c3499a56c1906"
+  end
+  if OS.linux? && Hardware::CPU.arm? && Hardware::CPU.is_64_bit?
+    url "https://github.com/stacc/stacc-CLI/releases/download/v0.1.0/stacc_0.1.0_Linux_arm64.tar.gz"
+    sha256 "8180dc3a895bff235c5f5d9b44ccc3b1770763eb9e9ccd5fc8b239d726f00150"
+  end
+
+  def install
+    bin.install "stacc"
+  end
+
+  test do
+    system "#{bin}/stacc"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.